packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -85,11 +85,20 @@ contract DeployImplementationsInput is CommonBase {
         else revert("DeployImplementationsInput: unknown selector");
     }
 
+    function set(bytes4 sel, bytes32 _value) public {
+        if (sel == this.salt.selector) _salt = _value;
+        else revert("DeployImplementationsInput: unknown selector");
+    }
+
     function loadInputFile(string memory _infile) public pure {
         _infile;
         require(false, "DeployImplementationsInput: not implemented");
     }
 
+    function salt() public view returns (bytes32) {
+        return _salt;
```

Comment:
Is it worth checking the `_salt` is not zero? We do this for all over getters.

Reply:
Added a comment to clarify why we do not do this in 6bef54667—the zero salt is a valid salt